commit cf3832cd4124b29d7b959effe98c56e299f17f02 (refs/changes/19/1319/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-01-23T11:27:38-08:00 (2 years, 8 months ago)
    
    PUBAPI-1358 cloudapi's usage of sdc-clients' IMGAPI client is working by luck

diff --git a/package.json b/package.json
index c9ec288..720149a 100644
--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
         "bunyan": "1.8.1",
         "cueball": "1.1.9",
         "kang": "1.1.0",
-        "sdc-clients": "10.0.0",
+        "sdc-clients": "10.0.4",
         "ufds": "1.2.0",
         "semver": "2.2.1",
         "nodemailer": "0.3.29",
